core/src/main/java/org/infinispan/commands/tx/VersionedCommitCommand.java

```diff
@@ -20,14 +21,14 @@
    private EntryVersionsMap updatedVersions;
 
    public VersionedCommitCommand() {
-      super("");
+      super(ByteString.emptyString());
```

Comment:
Might as well use `null`, almost all commands use `super(null)` in the "CommandIdUniquenessTest constructor".